wx_test_case.hpp

```diff
@@ -23,9 +23,15 @@
 #define wx_test_case_hpp
 
 #include "config.hpp"
+#include "mvc_controller.hpp"
 
 #include <boost/filesystem/path.hpp>
 
+#include <wx/testing.h>
+#include <wx/filedlg.h>
```

Comment:
```suggestion
#include <wx/filedlg.h>
#include <wx/testing.h>
```
(should be in alphabetic order, strange that this hasn't been picked up by concinnity check)